integrationtest/chaincode/private/main.go

```diff
@@ -28,7 +29,7 @@ func (sc *PrivateContract) GetPrivateState(ctx contractapi.TransactionContextInt
 	}
 
 	if bytes == nil {
-		return "", errors.New("No value found for " + key)
+		return "", fmt.Errorf("No value found for " + key)
```

Comment:
This is not appropriate usage of fmt.Errorf since the format specifier does not contain any formatting verbs, and no additional arguments are supplied. It should either be rewritten using fmt.Errorf and formatting verbs or left as-is with an explicitly concatenated string passed to errors.New.